core/domain/classifier_domain.py

```diff
@@ -332,6 +332,11 @@ def validate(self):
                 feconf.ALLOWED_TRAINING_JOB_STATUSES,
                 self.exp_version)
 
+        if not isinstance(self.interaction_id, basestring):
```

Comment:
We can do more than this, right? I.e. check the interaction id against our list of known interaction IDs, similar to what we do in state validation?